app/reducers/reducers.jsx

```diff
@@ -1,7 +1,7 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from '../app';
+import { combineReducers } from 'redux';
```

Comment:
This file should be `import { default as dashboard } from './dashboardReducers';`